common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1553,7 +1553,7 @@ def get_course_successors(self, course_locator, version_history_depth=1):
             next_versions = [struct for struct in next_entries]
```

Comment:
struct will be set to last entry of next_entries in python2. So changed to next_entries[-1:][0]['_id'] instead of struct['_id'] as struct scope is limited to comprehension only.

Reply:
Why not `next_entries[-1]` ?